gramex/handlers/mlhandler.py

```diff
@@ -367,7 +385,16 @@ def _train(self, data=None):
         data = self._filtercols(data)
         data = self._filterrows(data)
         self.model = self._assemble_pipeline(data, force=True)
-        if not isinstance(self.model[-1], TransformerMixin):
+        if isinstance(self.model, BaseModel):
+            index = self.get_argument('index_col')
+            if data.index.name != index and index in data:
+                data[index] = pd.to_datetime(data[index])
+                data.set_index(index, verify_integrity=True, inplace=True)
+            result = _fit(
+                self.model, None, data, self.model_path, target_col=target_col
+            ).res.summary().as_html()
+            self.set_header('Content-Type', 'text/html')
+        elif not isinstance(self.model[-1], TransformerMixin):
```

Comment:
There has to be a more elegant way of checking if the estimator at the end of a pipeline, if it is a pipeline, is a transformer or not, _without_ needing the model object to be subscriptable. If nothing else, `mclass` exists in config.json or gramex.yaml.